core/trino-main/src/test/java/io/trino/type/TestRowOperators.java

```diff
@@ -304,11 +304,11 @@ public void testJsonToRow()
                 asList("puppies", "[1,2,3]", null, "null"));
 
         assertFunction(
-                "CAST(JSON '{\"varchar_value\": \"puppies\", \"json_value\": [1, 2, 3], \"varchar_null\": null, \"json_null\": null}' " +
-                        "AS ROW(varchar_value VARCHAR, json_value JSON, varchar_null VARCHAR, json_null JSON))",
+                "CAST(JSON '{\"varchar_value\": \"puppies\", \"json_value_field\": [1, 2, 3], \"varchar_null\": null, \"json_null\": null}' " +
```

Comment:
The problematic occurrence is the field name in the row type constructor. I could quote `json_value` there, but chose to rename the field instead, so that all row field names remain unquoted. Do you think it's better to quote?

Reply:
Oh - I understand now. Leave as is.